pretest.js

```diff
@@ -0,0 +1,47 @@
+var async = require('async');
+var fs = require('fs');
+var settings = require('./tests-settings.js');
+var oracle = require('./')(settings);
+var assert = require('assert');
+
+var raw = fs.readFileSync('./test/test.sql', 'utf8');
+var statements = [];
+var stmt = '';
+// split test/test.sql file into statements, but strip any trailing ;s unless
+// they are at the end of an indented line, wich indicates a substatement.
+raw.split('\n').forEach(function(line) {
+  if (/\//.test(line) || /^\s+$/.test(line)) {
+    return;
+  }
+  if (/^\S+.*;$/.test(line)) {
+    if (!/^end;$/i.test(line)) {
+      stmt += line.slice(0, -1);
+    } else {
+      stmt += line;
+    }
+    statements.push(stmt.trim());
+    stmt = '';
+    return;
+  }
+  stmt += line + '\n';
+});
+
+oracle.connect(settings, function (err, connection) {
+  if (err) {
+    console.error(err);
```

Comment:
I suppose that has the added advantage of making the tests bail early if a DB can't be connected to.